nikola/plugins/compile/markdown/__init__.py

```diff
@@ -43,40 +44,56 @@
 from nikola.utils import makedirs, req_missing, write_metadata
 
 
+class ThreadLocalMarkdown(threading.local):
+    """Convert to markdown using per-thread Markdown objects."""
+
+    def __init__(self, extensions):
+        self.markdown = Markdown(extensions=extensions, output_format="html5")
```

Comment:
Yep, I'm happy with this PR. I'm just not happy how `threading.local` was designed ;)